# smart_home/core/erros.py: TransicaoInvalida, ConfigInvalida, ValidacaoAtributo.
class SmartHomeError(Exception): ...
class DispositivoJaExiste(SmartHomeError): ...
class DispositivoNaoEncontrado(SmartHomeError): ...
class ComandoInvalido(SmartHomeError): ...
class AtributoInvalido(SmartHomeError): ...
class ConfigInvalida(SmartHomeError): ...
